### ISSUE TITLE
Shortcuts do not respect DefaultKeyBinding.dict ?

### ISSUE THREAD

**Issue by Kirow** [xi-editor/xi-mac#171]
I have custom `~/Library/KeyBindings/DefaultKeyBinding.dict` file (Windows user habit):
```
{
    "\UF729"  = moveToBeginningOfParagraph:; // home
    "\UF72B"  = moveToEndOfParagraph:; // end
    "$\UF729" = moveToBeginningOfParagraphAndModifySelection:; // shift-home
    "$\UF72B" = moveToEndOfParagraphAndModifySelection:; // shift-end
}
```

**Home** / **End** works fine, but **Shift-Home** / **Shift-End** - not. I hear error sound instead. Should work same way as **CMD-Shift-Left** / **CMD-Shift-Right**

**Comment by raphlinus** [comment]
These bindings are simply missing (see https://github.com/google/xi-mac/blob/master/XiEditor/EditViewController.swift#L290). It would be a fairly straightforward patch to add them; probably would require a patch to both front-end and core.

**Comment by terhechte** [comment]
I'll give this a go

**Comment by jansol** [comment]
Fixed in xi-editor/xi-editor#892 and #292
